src/LiveStorage.ts

```diff
@@ -5,7 +5,10 @@ interface BroadcastChannelMessageType<ValueType> {
   }
 }
 
-const IS_BROWSER = typeof window !== 'undefined'
+// Jest jsdom contains `window` object in Node => detect required features
```

Comment:
That's a good catch. Does Jest polyfill `BroadcastChannel`? I'd expect the `sessionStorage` to be polyfilled.

Reply:
BroadcastChannel  was undefined in Jest (both node and jsdom environments), I haven't tried sessionStorage what would happen if I try to set+get a value yet

Reply:
Great, I love this approach.